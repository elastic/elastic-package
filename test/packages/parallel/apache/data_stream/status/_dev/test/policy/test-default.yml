vars:
  hosts:
    - http://localhost:8080
