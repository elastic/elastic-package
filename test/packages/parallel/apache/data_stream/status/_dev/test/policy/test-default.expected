inputs:
    - data_stream:
        namespace: ep
      meta:
        package:
            name: apache
      name: test-default-apache
      streams:
        - data_stream:
            dataset: apache.status
            type: metrics
          hosts:
            - http://localhost:8080
          metricsets:
            - status
          period: 30s
          server_status_path: /server-status
      type: apache/metrics
      use_output: default
output_permissions:
    default:
        _elastic_agent_checks:
            cluster:
                - monitor
        _elastic_agent_monitoring:
            indices: []
        uuid-for-permissions-on-related-indices:
            indices:
                - names:
                    - metrics-apache.status-ep
                  privileges:
                    - auto_configure
                    - create_doc
secret_references: []
